integration_tests/src/main/python/array_test.py

```diff
@@ -307,16 +307,20 @@ def do_it(spark):
     assert_gpu_and_cpu_are_equal_collect(do_it)
 
 # TODO add back in string_gen when https://github.com/rapidsai/cudf/issues/9156 is fixed
```

Comment:
Is this comment outdated? The linked issue has been resolved and it looks like `string_gen` has been added back to this list.

Reply:
I guess we could remove the comment. But it is not related to this PR.